packages/connect-plugin-stellar/package.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "@trezor/connect-plugin-stellar",
+    "version": "9.0.0",
+    "author": "Trezor <info@trezor.io>",
+    "homepage": "https://github.com/trezor/trezor-suite/packages/connect-plugin-stellar",
+    "description": "@trezor/connect plugin for Stellar",
+    "license": "SEE LICENSE IN LICENSE.md",
+    "repository": {
+        "type": "git",
+        "url": "git://github.com/trezor/trezor-suite.git"
+    },
+    "bugs": {
+        "url": "https://github.com/trezor/trezor-suite/issues"
+    },
+    "keywords": [
+        "Trezor",
+        "trezor-connect",
+        "stellar"
+    ],
+    "sideEffects": false,
+    "main": "index.js",
+    "peerDependencies": {
+        "stellar-sdk": "^10.1.0"
+    },
+    "devDependencies": {
+        "stellar-sdk": "^10.1.0"
+    },
+    "scripts": {
+        "test:unit": "jest",
+        "type-check": "tsc --build tsconfig.json",
+        "build:lib": "rimraf ./lib && tsc --build tsconfig.lib.json"
```

Comment:
add missing lint script for consistency with other packages
"lint": "eslint '**/*.{ts,js}'",

Reply:
added